scripts/bench.sh

```diff
@@ -5,9 +5,16 @@ MYDIR=$(realpath "$(dirname "$0")")
 cd "$MYDIR"
 cd ..
 
-NIGHTLY_TOOLCHAIN=${NIGHTLY_TOOLCHAIN:-nightly}
+NIGHTLY_TOOLCHAIN=${NIGHTLY_TOOLCHAIN:-nightly-2024-01-01}
 cargo +"$NIGHTLY_TOOLCHAIN" build --release --target wasm32-unknown-unknown -Z build-std=std,panic_abort -Z build-std-features=panic_immediate_abort
 
 export RPC_URL=http://localhost:8547
-cargo run --release -p benches
+
+# No need to compile benchmarks with `--release`
+# since gas measurement happens on the contract side
```

Comment:
I think this comment can be a bit clearer. Not sure if it means that the gas happens from outisde the node or what?

Reply:
The comment was like that [before](https://github.com/OpenZeppelin/rust-contracts-stylus/pull/254#discussion_r1771079049). @alexfertel offered to slightly improve it

Reply:
If I understand correctly the reason we don't need the `--release`flag is because this is running the benchmark infrastructure and not the smart contracts that are being measured? If that's the case may I offer an improved wording?
```suggestion
# since this only runs the benchmarking code and the contracts have already been compiled with `--release`
```

Reply:
beautiful! I'm so bad with words 😅